src/pages/management/configuration-api/v2.0/index.mdx

```diff
@@ -2384,6 +2384,7 @@ And here's a list of the greeting rules that you can use:
 | `search_keyword`    | When a visitor has searched for a specific keyword                                                                  |
 | `custom_variable`   | The greeting will be sent when the specified variable is met [documented below](#custom-variable)                   |
 | `url_funnel`        | The visitors who visit the specified sequence of websites will receive the greeting [documented below](#url-funnel) |
+| `ads_traffic`       | The greeting will be sent when customer fist visited page contains ad provider parameters                           |
```

Comment:
I am not sure what we want to achieve here🤔 It is not clear to me when I read it. Do we want?
1. The greeting will be sent when the customer's first visited page contains ad provider parameters
or
2. The greeting will be sent when the customer first visits the page that contains ad provider parameters
I assume we want option 1, but I prefer to confirm😜

Reply:
Option 2 seems fine